charts/seed-bootstrap/charts/extensions/templates/crd-dnsrecord.yaml

```
@@ -0,0 +1,281 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.0
+  creationTimestamp: null
```

Please add the following label, otherwise the CRD will not be deletion protected
```suggestion
  labels:
    gardener.cloud/deletion-protected: "true"
```

because there is an object selector in the GSAC webhook config:
https://github.com/gardener/gardener/blob/712a875b5965c9c87a8ce9c304c13733fe4b11ce/pkg/operation/botanist/component/seedadmissioncontroller/seedadmissioncontroller.go#L313-L315

We should add the same object select to the webhook config in the envtest, so that it can catch such issues: https://github.com/gardener/gardener/blob/ce60b9be18f869e0fa7917c91526072d0ca66a1e/pkg/seedadmissioncontroller/seedadmission_suite_test.go#L113
(or even better: reuse the webhook config from `pkg/operation/botanist/component/seedadmissioncontroller`)

---

Added the label, I believe the rest is not for this PR.